.github/workflows/ci-env-setup.yml

```
@@ -0,0 +1,39 @@
+    runs-on: ubuntu-latest
+    outputs:
+      env_name: ${{ steps.load-env-file.outputs.env_name }}
+      env_vars: ${{ steps.load-env-file.outputs.env_vars }}
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      
+      - name: 環境設定ファイル読み込み
+        id: load-env-file
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:6:15: Quote this to prevent word splitting

(shellcheck)